templates/profile.html

```diff
@@ -389,19 +389,19 @@ <h3>{{ username }}</h3>
                 <button type="submit" class="update-photo-btn">Update Photo</button>
             </form>
             <div class="container">
-                <!-- <form action="{{ url_for('change_username') }}" method="POST">
+                <!-- <form action="{{ url_for('change_username.change_username') }}" method="POST">
                     <label>New Username:</label>
                     <input type="text" name="new_username" required>
                     <button type="submit" class="formbutton">Update Username</button>
                 </form> -->
             
-                <form action="{{ url_for('change_email') }}" method="POST">
+                <form action="{{ url_for('change_email.change_email') }}" method="POST">
```

Comment:
A weirdly redundant method. Isn't it?

change_email.change_email, change_password.change_password, and change_username.change_username are all appearing redundant. If you are modularizing, look how these method names should be altered.

perhaps, email.change(), rather than change_email.change_email()?